neo/SmartContract/InteropService.NEO.cs

```diff
@@ -60,7 +63,82 @@ private bool Native_Deploy(ApplicationEngine engine)
             return true;
         }
 
-        private bool Header_GetVersion(ApplicationEngine engine)
+        private static bool Crypto_CheckSig(ApplicationEngine engine)
+        {
+            byte[] pubkey = engine.CurrentContext.EvaluationStack.Pop().GetByteArray();
+            byte[] signature = engine.CurrentContext.EvaluationStack.Pop().GetByteArray();
+
+            try
+            {
+                engine.CurrentContext.EvaluationStack.Push(Crypto.Default.VerifySignature(engine.ScriptContainer.GetHashData(), signature, pubkey));
+            }
+            catch (ArgumentException)
```

Comment:
In order to prevent future problems if we throw other exceptions, i prefer to return false for all of them

```suggestion
            catch
```

Reply:
`ArgumentException` means that the validation failed, so push `false`. Other exceptions should be thrown directly.